services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/converter/OfPortDescConverter.java

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.floodlight.converter;
+
+import org.openkilda.messaging.info.switches.PortDescription;
+
+import org.projectfloodlight.openflow.protocol.OFPortConfig;
+import org.projectfloodlight.openflow.protocol.OFPortDesc;
+import org.projectfloodlight.openflow.protocol.OFPortFeatures;
+import org.projectfloodlight.openflow.protocol.OFPortState;
+
+public final class OfPortDescConverter {
+
+    /**
+     * Convert {@link OFPortDesc} to format that kilda supports.
```

Comment:
"format that kilda supports." means a lot. Can we just mention a class or a payload for a specific response?